src/transformers/utils/fx.py

```diff
@@ -634,7 +703,7 @@ def check_has_proxy(v):
         else:
             return target(*args, **kwargs)
 
-    return wrapper, target
+    return wrapper
```

Comment:
I would rather keep the previous implementation as this is used in other packages. Maybe we should make this function public in that aspect.

Reply:
hum ... this function is prefixed with `_` ... When you say other packages do you know which one? Seems like you could very easily make the change?